### PR TITLE
appcenter-file-upload-client-node version to 1.1.1

### PR DESCRIPTION
**Pull request by anywherepilot**
Dependency updates only

### CODE REVIEW

**Review by jp-andre — APPROVED** [review]

File: appcenter-file-upload-client-node/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "appcenter-file-upload-client-node",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (ajaykn) [inline appcenter-file-upload-client-node/package.json:3]: is this some bug fix ?

Comment (anywherepilot) [reply appcenter-file-upload-client-node/package.json:3]: Dependency updates. We only change the patch version for that.

Comment (jp-andre) [reply appcenter-file-upload-client-node/package.json:3]: Dependency updates only

Comment (ajaykn) [reply appcenter-file-upload-client-node/package.json:3]: ah cool

**Pull request merged**